ALUM/ALUM/Services/PostSessionService.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  PostSessionService.swift
+//  ALUM
+//
+//  Created by Adhithya Ananthan on 4/26/23.
+//  This file will contain the service function to make a post session route
+//  Note that the function in this file should be added to the SessionService page
+//  once everything is merged
+//
+
+import Foundation
+
+
+struct SessionPostData: Codable {
+    var menteeId: String
+    var mentorId: String
+    var calendlyURI: String
+}
+
+struct PostSessionData: Codable {
+    var sessionId: String
+    var menteeId: String
+    var mentorId: String
+}
+
+class PostSessionService {
+    func postSessionWithId(menteeId: String, mentorId: String, calendlyURI: String) async throws -> PostSessionData?  {
+        let urlObj = URL(string: "http://localhost:3000/sessions")!
+        var request = URLRequest(url: urlObj)
+        
+        /*
+        guard let authToken = try await UserService().getCurrentAuth() else {
+                print("Could not get auth token")
+                throw APIError.invalidRequest(message: "Could not get auth token")
+        }
+         */
```

Comment:
Good idea for testing but let's integrate this with the auth token stuff before merge